server/controllers/profile.js

```diff
@@ -41,3 +41,18 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
     },
   });
 });
+
+// @route GET /profile/listing
+// @desc Get sitter profile data
+// @access Private
+exports.loadSitters = asyncHandler(async (req, res, next) => {
+  const profile = await Profile.find({ sitter: true });
+  if (!profile) {
```

Comment:
We don't need this check here. 
Also, since we've added the `protect` middleware to this route - unauthorized users cannot access it